Calendars.Plugin.nuspec

```diff
@@ -15,9 +15,9 @@
        <description>
          Perform basic CRUD calendar operations on iOS/Android/WinPhone/UWP through a simple common API.
          
-See project site README for usage notes and limitations.
+         See project site README for usage notes and limitations.
```

Comment:
Is there a reason you think this indenting should be added? I specifically removed it before because it looks weird in Visual Studio otherwise.

![image](https://cloud.githubusercontent.com/assets/7707864/13101347/98ef47ba-d4fa-11e5-897a-583d2f9e868e.png)

(okay that was pretty awesome I did not actually expect pasting an image to work :) )